packages/phpstan-latte-rules/src/Rules/LatteCompleteCheckRule.php

```diff
@@ -171,7 +169,8 @@ private function processTemplateFilePath(
             return [$ruleError];
         }
 
-        $tmpFilePath = sys_get_temp_dir() . '/' . md5($scope->getFile()) . '-latte-compiled.php';
+        $serializedRenderTemplateWithParameters = serialize($renderTemplateWithParameters);
```

Comment:
If one file contains more templates with parameters, only last was used in analyse